cypress/support/common/verify-page-heading.js

```diff
@@ -1,9 +1,9 @@
-import {getPageHeading} from "../page-objects/common-PO";
-
-export const verifyPageHeading = (pageHeading) => {
-    getPageHeading()
-        .invoke('text')
-        .then((text) => {
-            expect(text).to.contain(pageHeading);
-        });
-};
+// import {getPageHeading} from "../page-objects/common-PO";
```

Comment:
Please un comment this function

Reply:
updated and removed some methods that are unused